### PR TITLE
Clean up vx-scan/backend

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview
Task: #2813 

Remove central scanner code from vx-scan/backend (which was a copy-paste of services/scan in #2816). I tried to just focus on removing and not factoring out duplicated code that both vx-scan/backend and services/scan depend on. Will return to that in a follow up.

Note that I removed some scripts that for now will live only in services/scan, but can be moved to a shared location later if desired.

_Review by commits_

## Demo Video or Screenshot
N/A

## Testing Plan 
- Automated tests
- Can still run/lint

## Checklist
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [ ] I have added JSDoc comments to any newly introduced exports

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: apps/vx-scan/backend/README.md [hunk 1]
```diff
@@ -76,45 +33,41 @@ MOCK_SCANNER_HTTP=1 pnpm start
...
-```
-
 #### Letter-sized ballots
 
-First init `services/smartcards` with:
+First configure the scanner with the election definition by running
+`services/smartcards` with:
 
 ```
 ./mockCardReader.py enable --admin ../scan/test/fixtures/choctaw-2020-09-22-f30480cc99/election.json
```
Comment (kofi-q) [inline apps/vx-scan/backend/README.md:42]: s/scan/vx-scan

**Review by kofi-q — APPROVED** [review]
✂️ ✂️

File: apps/vx-scan/backend/README.md (hunk 1 above)
Comment (jonahkagan) [reply apps/vx-scan/backend/README.md]: whoops, nice catch dcc8652cd

**Review by arsalansufi — COMMENTED** [review]
This got merged before I was able to finish my low-grade photoshop 😆

mr-clean-lgtm

File: apps/vx-scan/backend/package.json [hunk 2]
```diff
@@ -94,24 +85,19 @@
...
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-vx": "workspace:*",
-    "fast-check": "^2.23.2",
     "is-ci-cli": "^2.1.2",
     "jest": "^26.6.3",
     "jest-junit": "^14.0.1",
     "jest-watch-typeahead": "^0.6.4",
     "lint-staged": "^10.5.3",
     "nock": "^13.1.0",
-    "nodemon": "^2.0.20",
```
Comment (eventualbuddha) [inline apps/vx-scan/backend/package.json:114]: This was removed too aggressively and `pnpm start` in the `frontend` no longer works.

Comment (adghayes) [reply apps/vx-scan/backend/package.json:114]: I am running in to this issue on my branch.

Comment (jonahkagan) [reply apps/vx-scan/backend/package.json:114]: Sorry about that! Fixed in #2824

**Pull request merged**